Misc/NEWS.d/next/Build/2020-02-28-23-51-47.bpo-38380.u-ySyA.rst

```diff
@@ -0,0 +1 @@
+Update macOS builds to use SQLite 3.31.1
```

Comment:
Move this to the `macOS` section (see [previous entries](https://docs.python.org/3/whatsnew/changelog.html#id73))

Reply:
Fixed in commit 6a89f6489bc9bfd3c8c31d677f4a8c3fc59f5b0e